Tests/merge_test.py

```diff
@@ -233,8 +233,7 @@ def test_GSUB_merge_with_different_dups_and_no_second_GSUB(self):
 
 		table = self.mergedTable.table
 		self.assertEqual(table, self.table1.table)
-		self.assertTrue(len(captor.records) > 0)
-
+		self.assertTrue(len([r for r in captor.records if r.msg != '']) > 0)
```

Comment:
@zjusbo sorry, I'm drunk.. forget the last message

This is what I actually meant:

```python
self.assertTrue(len([r for r in captor.records if "is dropped and replaced by" in r.msg]) > 0)
```